sirepo/job_supervisor.py

```diff
@@ -173,6 +175,24 @@ def cache_timeout_set(self):
             self.cache_timeout,
         )
 
+    def clear_status(self, op, exception=None):
+        if not exception:
+            self.__db_update(jobStatusMessage=None)
+            return
+        if not isinstance(exception, asyncio.CancelledError):
+            # All non CancelledError exceptions should be preserved in the db
+            # for future debugging
+            return
+        # If we initiated the cancel then clear the jobStatusMessage
+        # TODO(e-carlin): Race condition - Another run may have started so we
```

Comment:
@robnagler any thoughts on how you want to handle this? I'm inclined to just get rid of the exception handling code altogether. In the case of an exception (cancelledError or not) we just leave the jobStatusMessage in the db.

Reply:
There is another race condition. If the runSim deletes the self._cancelledSerial before we get a chance to use it then we also won't remove the jobStatusMessage. This pushes me to be even more in favor of just leaving it on any exception.

Reply:
I think we shouldn't clear_status. It's only used for auditing. We can know, for example, that something was waiting for a cpu_slot when it was canceled.